validator/schema/releases_schema.py

```diff
@@ -0,0 +1,111 @@
+from schema import Schema, Optional, And, Or, Regex, SchemaError
+from validator.schema.image_schema import IMAGE_CONTENT_SCHEMA
+from validator.schema.rpm_schema import RPM_CONTENT_SCHEMA
+
+GIT_SSH_URL_REGEX = r'((git@[\w\.]+))([\w\.@\:/\-~]+)(\.git)(/)?'
+
+ASSEMBLY_DEPENDENCIES = {
+    'rpms': [
+        And({
+            Regex(r'el\d+'): str,  # Each RHEL version can have its own
+            'why': str,  # Human description of why this is being added for historical purposes
+            'non_gc_tag': str,  # Artist should provide tag they know will prevent this NVR from garbage collection.
+        })
+    ]
+}
+
+ASSEMBLY_NAME_REGEX = Regex(r'^art\d+$')
+
+
+def releases_schema(file):
+    return Schema({
+        'releases': {
+            Optional(Or('stream', 'test', ASSEMBLY_NAME_REGEX)): {
+                'assembly': {
+                    Optional('basis'): {
+                        Optional('brew_event'): int,
+                        Optional('assembly'): ASSEMBLY_NAME_REGEX,
+                    },
+                    Optional('rhcos'): {
+                        'machine-os-content': {
+                            'images': {
+                                # pullspecs for arch specific images
+                                Optional('x86_64'): str,
+                                Optional('s390x'): str,
+                                Optional('ppc64le'): str,
+                                Optional('arm64'): str,
```

Comment:
`arm64` is golang arch name while `x86_64` is gcc arch name. We can tolerant both naming styles but it's better to use a consistent style here. Suggest `s/arm64/aarch64`.

Reply:
@vfreex , the golang arch name of what we call `x86_64` is `amd64`, not `arm64`, right?

Reply:
but I see what you are saying, since we don't use the golang arch name for `x86_64`, why should we use it for ARM.
|golang|ART|
|---------|------|
|amd64|x86_64|
|arm64|aarch64|